src/QRPlatba.php

```diff
@@ -25,7 +25,7 @@ class QRPlatba
         'ACC'     => null, // Max. 46 - znaků IBAN, BIC Identifikace protistrany !povinny
         'ALT-ACC' => null, // Max. 93 - znaků Seznam alternativnich uctu. odddeleny carkou,
         'AM'      => null, //Max. 10 znaků - Desetinné číslo Výše částky platby.
-        'CC'      => 'CZK', // Právě 3 znaky - Měna platby.
```

Comment:
CZK should remain here for backward compatibility. Please edit this, then I will merge and release.